react/features/mobile/navigation/components/RootNavigationContainer.js

```diff
@@ -35,50 +34,51 @@ type Props = {
 }
 
 
-const RootNavigationContainer = ({ isWelcomePageAvailable }: Props) => (
-    <SafeAreaProvider>
-        <NavigationContainer
-            independent = { true }
-            ref = { rootNavigationRef }
-            theme = { navigationContainerTheme }>
-            <RootStack.Navigator
-                initialRouteName = { screen.root }>
-                {
-                    isWelcomePageAvailable
-                        ? <RootStack.Screen
-                            component = { WelcomePageNavigationContainer }
-                            name = { screen.root }
-                            options = { drawerNavigatorScreenOptions } />
-                        : <RootStack.Screen
-                            component = { BlankPage }
-                            name = { screen.root }
-                            options = {{
-                                gestureEnabled: false,
-                                headerShown: false
-                            }} />
-                }
-                <RootStack.Screen
-                    component = { DialInSummary }
-                    name = { screen.dialInSummary }
-                    options = { dialInSummaryScreenOptions } />
-                <RootStack.Screen
-                    component = { LoadConfigOverlay }
-                    name = { screen.loadConfigOverlay }
-                    options = {{
-                        gestureEnabled: false,
-                        headerShown: false
-                    }} />
-                <RootStack.Screen
-                    component = { ConferenceNavigationContainer }
-                    name = { screen.conference.root }
-                    options = {{
-                        gestureEnabled: false,
-                        headerShown: false
-                    }} />
-            </RootStack.Navigator>
-        </NavigationContainer>
-    </SafeAreaProvider>
-);
+const RootNavigationContainer
+    = ({ isWelcomePageAvailable }: Props) => (
+        <SafeAreaProvider>
+            <NavigationContainer
+                independent = { true }
+                ref = { rootNavigationRef }
+                theme = { navigationContainerTheme }>
+                <RootStack.Navigator
+                    initialRouteName = { screen.root }>
+                    {
+                        isWelcomePageAvailable
+                            ? <RootStack.Screen
+                                component = { WelcomePageNavigationContainer }
+                                name = { screen.root }
+                                options = { drawerNavigatorScreenOptions } />
+                            : <RootStack.Screen
```

Comment:
I know it's not technically part of this PR, but I think it makes sense to tackle both problems at once: we should make connecting be the first route to avoid SDK users seeing a "Meeting ended" flashing.